src/ai/backend/kernel/intrinsic.py

```diff
@@ -148,9 +148,16 @@ async def prepare_sshd_service(service_info):
 
 async def prepare_ttyd_service(service_info):
     shell = "sh"
-    if Path("/bin/bash").exists():
+    if Path("/bin/zsh"):
+        shell = "zsh"
+        zshrc_path = Path("~/.zshrc").expanduser()
+        with zshrc_path.open("w") as f:
+            f.write("export PATH=$PATH:/opt/backend.ai/bin\n")
+            f.write("export TERM=xterm-256color\n")
```

Comment:
I think we are already providing the `TERM` env-var with the value "xterm".
Is there any extra features in zsh that requires this?

Reply:
In that case we can remove this line. The main purpose was to add colors to the shell